app/jobs/usps_auth_token_refresh_job.rb

```diff
@@ -0,0 +1,38 @@
+class UspsAuthTokenRefreshJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    analytics.idv_usps_auth_token_refresh_job_started
+
+    if token_expiration < 7.minutes
+      usps_proofer.retrieve_token!
+    end
+
+    return true
```

Comment:
If we wanted to have an explicit return value, I would use ruby's implicit return here:

However I think we should not return a value at all (because its a background job, and nothing will receive that value). I think a previous iteration of this PR used the explicit return in specs but looks like we're not doing that here (which is better IMO) so we can just remove

```suggestion
```

Reply:
Thanks for catching that!